brainstormforce/astra
# AST-1674 - Astra's Toggle menu does not work on iOS devices below 13 version.

### Description
Astra's Toggle menu does not work on iOS 12 or iOS 11 versions.

### Screenshots
- https://share.bsf.io/RBundQOQ

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
*How to reproduce*
1. Add a toggle menu to the website
2. Test the website on iOS 12 or iOS 11
3. Check on iOS 13 or higher, it works fine. 

### Checklist:
